_install-and-configure/additional-plugins/analysis-phonenumber-plugin.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Phone number analysis plugin
+parent: Installing plugins
+nav_order: 20
+
+---
+
+# Phone number analysis plugin
+Introduced 2.18
+{: .label .label-purple }
+
+The `analysis-phonenumber` plugin provides analysers and tokenizers for phone numbers.
+A dedicated analyser is required since parsing phone numbers is a non-trivial task (even though it might seem trivial at
+first glance!), as can be seen in the list [falsehoods programmers believe about phone numbers](https://github.com/google/libphonenumber/blob/master/FALSEHOODS.md).
+
+## Installing the plugin
+
+Install the `analysis-phonenumber` plugin using the following command:
+
+```sh
+./bin/opensearch-plugin install analysis-phonenumber
```

Comment:
@reta: i presume this will work since the plugin should be built & published automatically, right?

Reply:
Yes :100%